styles.css

```diff
@@ -358,11 +366,75 @@ ul {
   color: #f55800;
 }
 
-hr {
+#firstlin {
   margin-top: 180px;
   width: 80%;
 }
 
+footer {
+  background: #fff;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 120px 24px;
+  gap: 58px;
+}
+
+#foot-text {
+  width: 327px;
+  height: 216px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+form {
+  width: 327px;
+  height: 330px;
+}
+
+#form-ul {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  width: 327px;
+  height: 330px;
+  list-style-type: none;
+}
```

Comment:
Kindly stick to the design as much as possible. There are some disparities between your design and that of the figma especially with regards to the spacing and positioning. There is a blank space to the right side which increases with increasing screen width as shown below. :1234: Kindly check and fix that.

Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-01 15-46-31](https://user-images.githubusercontent.com/65233860/199276106-c26a7a47-acfb-4f72-ba26-1254b15b4ded.png)  |  ![Screenshot from 2022-11-01 15-46-42](https://user-images.githubusercontent.com/65233860/199276137-676b2d2b-6657-4cf9-aade-0b0297250027.png)